deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -34,15 +34,23 @@ spec:
             cpu:
               description: CPU defines set of CPU related parameters.
               properties:
+                enableIsolcpus:
+                  description: EnableIsolcpus results in the Isolated CPUset being
+                    listed in the isolcpu kernel arg on the host machines. When this
+                    option is set to "false", the Isolated cpus will be partitioned
+                    by tuned, but load balancing across the Isolated cpus will still
+                    be possible. When this option is set to "true", the Isolated CPU
```

Comment:
@davidvossel   In case we had the full cpu_partitioning profile support from NTO which also includes: no_balance_cores that disable the kernel load balancing in certain isolated CPUs, could we then say that this boolean will becomes redundant? (see [here](https://github.com/redhat-performance/tuned/blob/bfe0eb91949d4e305de5c22bec913fc8318b3a33/man/tuned-profiles-cpu-partitioning.7#L55))

Reply:
Actually, this is pretty good reason to keep the argument. We can then always pass `isolated` to isolated_cores=<CPU\-LIST> and iff the boolean is set then set the same list to no_balance_cores as well (isolcpus for now).